packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -140,6 +142,12 @@ export const Product = ({
     [setCreateTokenError]
   );
 
+  // clear any error rendered with `ErrorMessage` on form change
+  const onChangeErrorDismiss = useCallback(() => {
+    setCreateTokenError({ type: '', error: false });
+    resetCreateSubscriptionError();
+  }, [setCreateTokenError, resetCreateSubscriptionError]);
```

Comment:
I was thinking I could add a check here for each error message (if createTokenError, setCreateTokenError) but that didn't seem quite right, and now that I'm looking at this again, maybe it's better to pass in a new boolean prop like `displayErrorMessage` into `PaymentForm` that when true, clears all the possible errors. Otherwise this gets triggered on every `onChange` event in the form.